### PR TITLE
Enable AF support for the baseline -u config_url option

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
I think we should ignore th LGTM warning as we'll probably use it in the future...

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Sounds good to me.

File: docker/integration_tests/baseline_tests.sh [hunk 1]
```diff
@@ -82,7 +82,30 @@ fi
 rm ~/.ZAP_D/config.xml
 
 echo
-echo "TEST Baseline 4 (new vs old) - we expect _some_ differences and this will not fail the whole script"
+echo "Baseline test 4 (vs example.com with INFO/WARN/FAIL and OUTOFSCOPE set via URL)"
+/zap/zap-baseline.py -t https://www.example.com/ --auto -u https://raw.githubusercontent.com/zaproxy/zaproxy/main/docker/integration_tests/configs/baseline3.conf > /zap/wrk/output/baseline4.out
```
Comment (ricekot) [inline docker/integration_tests/baseline_tests.sh:86]: We don't need to specify `--auto` now right?

Comment (psiinon) [reply docker/integration_tests/baseline_tests.sh:86]: Good point, let me try that - it should work and if so I can remove that...

Comment (psiinon) [reply docker/integration_tests/baseline_tests.sh]: Yep, all worked, so removed them :)

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**